client/src/App.js

```diff
@@ -18,6 +20,13 @@ function App() {
         <Route path = '/user/id/events' element = {<UserEvents/>}/>
         <Route path = '/user/id/edit' element = {<UserEdit/>}/>
       </Routes>
+
+      <Routes>
+        <Route path ='/events' element={<Events/>}/>
+        <Route path ='/event/id' element = {<EventPage/>}/>
```

Comment:
A few nitpicks:
I think the standard convention is to keep the name as `/events/id` rather than `/event/id` to show that we're selecting one element from the results of events and someone could just add the id to the end of the path if they knew it to view the event rather than typing an entirely different route.

However, since this causes paths to overlap, we want to order our routes in decreasing specificity (write the second route before the first) so that a route like /events/2 matches the EventPage route rather than the Events route.

https://v5.reactrouter.com/web/example/url-params
For URL params in React, we need to prefix the path variable with a colon like `/events/:id` to be able to use the id as a variable later. 
```
const EventsPage = () => {
  let { id } = useParams();
  return <div></div>
}
```